bin/entrypoint

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+from pensieve.experimenter import ExperimentCollection
+import datetime as dt
+
+
+last_week = dt.datetime.now(dt.timezone.utc) - dt.timedelta(weeks=1)
+collection = ExperimentCollection.from_experimenter().started_since(last_week)
```

Comment:
Do we only want the experiments that started in the past week. Or experiments that were running in the last week (includes also the ones that potentially started a long time ago)?

Reply:
This is basically just to exercise the package and make sure it installed correctly – I have 0 attachement to the actual contents of that file :)

Reply:
Ah okay